components/tools/OmeroWeb/omeroweb/webgateway/urls.py

```diff
@@ -263,12 +263,42 @@
     - did:  Well ID
 """
 
+
 webgateway_plategrid_json = url(
     r'^plate/(?P<pid>[0-9]+)/(?:(?P<field>[0-9]+)/)?$',
     'webgateway.views.plateGrid_json', name="webgateway_plategrid_json")
 """
 """
 
+
+webgateway_get_thumbnails_json = url(
+    r'^get_thumbnails/(?:(?P<w>[0-9]+)/)?$',
+    'webgateway.views.get_thumbnails_json')
+"""
+Returns a set of thumbnail base64 encoded of the OMERO Images,
+optionally scaled to max-width and max-height.
+Params in get_thumbnails/<w>/ are:
+    - iid:  Image ID
```

Comment:
If I'm reading the code right, you request image ids using query string ?id=1&id=2 etc?
This could be explained a bit clearer here.
Also you mention max-height but only the option to set ```max width```.
Presumably ```max width``` is really ```longest side```?